test-harness/README.md

```diff
@@ -0,0 +1,36 @@
+## Generating binaries
+
+For anything to work, the binaries have to be generated first with: `yarn build.binaries`. This will generate three binaries.
+
+### Alternatively, run any of the following:
+
+1. `yarn build.binary` will assume the host machine
+2. `HOST=host yarn build.binary` will generate a binary for the current host machine
+3. `HOST=node10-linux-x64 yarn build.binary` will generate a binary for node10-linux-x64
+
+*Before running any of the above, make sure that `yarn` was run beforehand*. If `yarn` was not run, you might experience `Error: command mock not found` issue.
```

Comment:
So what we _really_ need is to make sure the dependencies are installed, and that can be done by either using `yarn` or `npm` — for this reason I'd rather say "make sure to have all the dependencies installed" rather than pointing to a specific command. Today yarn is a thing, maybe tomorrow it will not.